### PR TITLE
fix(splash): fix an error

### PR DESCRIPTION
**Pull request by Xzh0u**

### CODE REVIEW

File: players/g1_player.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def spread_points(current_point, angles: np.array, distance, reverse) -> np.arra
 def splash_zone(distance: float, angle: float, conf: float, skill: int, current_point: Tuple[float, float], in_sandtrap: bool) -> np.array:
     conf_points = np.linspace(1 - conf, conf, 5)
     scale = 1.1
-    if in_sandtrap: 
+    if in_sandtrap:
         # the distance rating is halved, and the standard deviations for the angle and distance distributions are doubled
         distances = np.vectorize(standard_ppf)(
             conf_points) * (distance / skill) * 2
```
Comment (yunlanli) [inline players/g1_player.py]: Did we miss + distance or + 2 * distance here?

When the standard deviation is not doubled, the formula for distance is:

```python
distances = np.vectorize(standard_ppf)(conf_points) * (distance / skill) + distance
```

Comment (Xzh0u) [reply players/g1_player.py]: There should be a distance / 2

Comment (Xzh0u) [reply players/g1_player.py]: It's weird, maybe I deleted it by mistake

Comment (yunlanli) [reply players/g1_player.py]: I'm not exactly sure what `distance / skill` represents, but based on the relationship that the distances from the sand trap should be halved, I think the formulas should be:

```python
distances = np.vectorize(standard_ppf)(conf_points) * 2 * (distance / skill) + distance / 2  # from sand trap
distances = np.vectorize(standard_ppf)(conf_points) *  (distance / skill) + distance  # from grass
```

What do you think?

File: players/g1_player.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def splash_zone(distance: float, angle: float, conf: float, skill: int, current_
     if in_sandtrap:
         # the distance rating is halved, and the standard deviations for the angle and distance distributions are doubled
         distances = np.vectorize(standard_ppf)(
-            conf_points) * (distance / skill) * 2
+            conf_points) * (distance / skill) * 2 + distance / 2
```
Comment (yunlanli) [inline players/g1_player.py:50]: > - The distance the ball actually travels is chosen by the simulator from N(d,d/s), the normal distribution with mean equal to d, the distance to the aimed point, and standard deviation d/s. The angle at which the shot is made may also differ from the aimed direction. If we take an angle of zero radians to mean directly towards the aiming point, then the actual angle will follow a normal distribution N(0,1/(2s)).
> - When hitting from a sand trap, the distance rating is halved, and the standard deviations for the angle and distance distributions are doubled.

looks good to me!

**Review by yunlanli — APPROVED** [review]
Looks good to me and the code runs on my computer.

File: players/g1_player.py (hunk 1 above)
Comment (Xzh0u) [reply players/g1_player.py]: I agree with you, and I already made some changes just now, the formula is the same as what you provided.

**Comment by abbybaldauf** [comment]
Looks good

**Pull request merged**